resources/event-hackathon/notebook-authoring-checklist.md

```diff
@@ -0,0 +1,120 @@
+    2. Sonifications
+    3. Interactive Widgets
+6. Related Resources
+
+## Organization
+
+At the beginning, a notebook has
+
+- [ ] A title in the form of a H1 (`#` in Markdown).
+- [ ] A brief summary of the notebook.
+- [ ] A table of contents in an ordered list (`1., 2.,` etc. in Markdown).
+- [ ] The author(s) and affiliation(s) (if relevant).
+- [ ] The date first published.
+- [ ] The date last edited (if relevant).
+- [ ] A link to the notebook's source(s) (if relevant).
+
+Throughout the notebook
+
+- [ ] There is only one H1 (`#` in Markdown) used in the notebook.
+- [ ] The notebook uses other heading tags in order (meaning it does not skip numbers).
+
+## Images
+
+- [ ] All images (jpg, png, svgs) have an image description. This could be
+    - [ ] Alt text (an `alt` attribute) 
+    - [ ] Empty alt text for decorative images/images meant to be skipped (an `alt` attribute with no value) or
+    - [ ] Captions or
+    - [ ] If no other options will work, the image is decribed in surrounding paragraphs.
+
+- [ ] Any text present in images exists in a text form outside of the image (this can be alt text, captions, or surrounding text.)
+
+### Visualizations
+
+- [ ] All visualizations have an image description. Review the previous section, Images, for more information on how to add it.
+- [ ] Visualization descriptions include
+    - [ ]  The type of visualization (like bar chart, scatter plot, etc.)
+    - [ ] Title
+    - [ ] Axis labels and range
+    - [ ] Key or legend
+    - [ ] An explanation of the visualization's significance to the notebook (like the trend, an outlier in the data, what the author learned from it, etc.)
+
+- [ ] All visualizations have the following labels
+    - [ ] Title
+    - [ ] Labels on all axes
+    - [ ] Key or legend (if relevant)
+
+- [ ] All visualizations and their parts have enough color contrast to be legible. Remember that transparent colors have lower contrast than their opaque versions.
+- [ ] All visualizations convey information with more than color-coding. Color may also be mixed with text labels, patterns, or icons to fulfill this.
+- [ ] All visualizations have an additional way for notebook readers to access the information. Linking to the original data, including a table of the data in the same notebook, or sonifying the plot are all options.
+
+## Text
+
+- [ ] All link text is descriptive. It tells users where they will be taken if they open the link.
+- [ ] Use plain language wherever possible.
+- [ ] All acronyms are defined at least the first time they are used. 
+- [ ] Field-specific/specialized terms are used when needed, but not excessively.
+- [ ] Text is broken into paragraphs and/or cells where relevant.
+- [ ] Text is in complete sentences where relevant.
+
+## Code
+
+- [ ] Code sections are introduced/explained before they appear in the notebook. This could be a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.
```

Comment:
Code sections are introduced and explained before they appear in the notebook. This can be fulfilled with a heading in a prior Markdown cell, a sentence preceding it, or a code comment in the code section.